### PR TITLE
Who says we can't be W3C compliant? Also, added Facebook Open Graph p…

### PR DESCRIPTION
**Pull request by Jac21**
…roperties for when the inevitable happens

### CODE REVIEW

**Review by dskoda1 — APPROVED** [review]

**Comment by dragondave** [comment]
Does that `&` need to be an `&amp;`?

**Comment by adeelx** [comment]
:+1:

**Comment by Jac21** [comment]
@dragondave 

Facebook handles the URLs in those properties quite nicely, and does any encoding necessary on their end once processed (I've seen this particular scenario handled before). It wouldn't hurt to play it safe, though.

File: server/index.html [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+		<meta property="og:url"                content="http://chaosthebot.com/" />
+		<meta property="og:type"               content="website" />
+		<meta property="og:title"              content="ChaosBot" />
+		<meta property="og:description"        content="A social coding experiment that updates its own code democratically" />
+		<meta property="og:image"              content="https://avatars3.githubusercontent.com/u/28798182?v=3&s=400" />
+	</head>
+	<body>
+		<p>Hello World!</p>
+	</body>
+</html>
```
Comment (shellandbull) [inline server/index.html]: missing newline here

**Review by shellandbull — CHANGES_REQUESTED** [review]

File: server/index.html (hunk 1 above)
Comment (Jac21) [reply server/index.html]: Noted, updating now. Thanks!

**Review by davidak — APPROVED** [review]

**Review by davidak — APPROVED** [review]

**Review by jetpacktuxedo — APPROVED** [review]

**Comment by Jac21** [comment]
image

**Pull request merged**